### ISSUE TITLE
Add README.txt or something for locally cloned wikis

### ISSUE THREAD

**Issue by spenserblack** [spenserblack/actions-wiki#48]
### Discussed in https://github.com/spenserblack/actions-wiki/discussions/46

<div type='discussions-op-text'>

<sup>Originally posted by **jcbhmr** March 22, 2023</sup>
I just had the random thought to include some kind of a file that would **not be visible on github** but would be if you cloned the wiki locally to say "heyyyy checkout the github repo instead, don't put stuff in the wiki here!"

similar idea i had just now while writing this would be to prepend a `<!-- DO NOT EDIT ON PUBLISHED WIKI, ONLY EDIT IN SOURCE REPO -->\n` or similar comment to all .md files to discourage editing the published version?

these ideas are kinda redundant if the user sees the "github-actions[bot] edited this" thing though... 🤷‍♀️
![image](https://user-images.githubusercontent.com/61068799/227085845-fb429b12-e16e-4a18-b41d-23e43efa6294.png)
</div>

**Comment by spenserblack** [comment]
This seems like a potentially valid idea, but right now it's not a high priority IMO, so I'm adding the "icebox" label. If this gets interest from the community, I'll remove the label.

**Comment by jcbhmr** [comment]
btw i think github projects was designed for this idea of "priority" stuff, while labels are intended to "categorize"?

maybe not tho cause here's a demo that uses projects to categorize. https://github.com/git-lfs/git-lfs/projects/3
but wait! here's the builtin template that is for priority (ish) since it uses a "backlog" and then other actively-working-on categories
![image](https://user-images.githubusercontent.com/61068799/227316227-ad549ebc-a85f-47df-bfa9-a872a44b62b2.png)

**Comment by spenserblack** [comment]
Yeah, I was considering creating a project, but then I felt that it was a bit of overkill for this repo. Right now a simple issue list is plenty IMO.

But, you're right, projects are great for this sort of thing :slightly_smiling_face:

**Comment by jcbhmr** [comment]
now that im thinking about it from like 3 weeks in the future; i think this is exactly what the ideas discussions are for! you can leave a discussion "open" and it's like a proposal idea in stasis for an _indefinite_ time (until a robot github workflow action closes it after 30 days or whatever if that's configured)

just a thought
